website/package.json

```diff
@@ -1,18 +1,77 @@
 {
   "name": "@verdaccio/website",
-  "version": "0.0.0",
+  "version": "5.1.2",
   "private": "true",
   "scripts": {
-    "examples": "docusaurus-examples",
-    "start": "docusaurus-start",
-    "build": "docusaurus-build",
-    "doc:version": "docusaurus-version",
-    "publish-gh-pages": "docusaurus-publish",
-    "write-translations": "docusaurus-write-translations",
-    "rename-version": "docusaurus-rename-version"
+    "docusaurus": "docusaurus",
+    "start": "docusaurus start",
+    "build": "docusaurus build",
+    "swizzle": "docusaurus swizzle",
+    "deploy": "docusaurus deploy",
+    "clear": "docusaurus clear",
+    "serve": "docusaurus serve",
+    "write-translations": "docusaurus write-translations",
+    "write-heading-ids": "docusaurus write-heading-ids",
+    "netlify:build:production": "docusaurus write-translations && pnpm -w crowdin:sync && docusaurus build",
+    "netlify:build:deployPreview": "docusaurus write-translations --locale en && docusaurus build --locale en",
+    "eslint:check": "eslint src/**/*.tsx",
+    "eslint:write": "eslint src/**/*.tsx --fix",
+    "prettier:check": "prettier src/**/* --check",
+    "prettier:write": "prettier src/**/* --write"
+  },
+  "engines": {
+    "node": ">=14"
+  },
+  "dependencies": {
+    "@docusaurus/core": "2.0.0-beta.3",
+    "@docusaurus/preset-classic": "2.0.0-beta.3",
+    "@docusaurus/remark-plugin-npm2yarn": "^2.0.0-beta.3",
+    "@docusaurus/theme-search-algolia": "^2.0.0-beta.3",
+    "@mdx-js/react": "^1.6.22",
+    "clsx": "^1.1.1",
+    "copy-text-to-clipboard": "^3.0.1",
+    "netlify-plugin-cache": "^1.0.3",
+    "prism-react-renderer": "^1.2.1",
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1",
+    "react-twitter-widgets": "^1.10.0",
+    "sass-loader": "^12.1.0",
+    "url-loader": "^4.1.1",
+    "use-is-in-viewport": "^1.0.9"
+  },
+  "browserslist": {
+    "production": [
+      ">0.5%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
   },
-  "license": "MIT",
   "devDependencies": {
-    "docusaurus": "^1.14.7"
+    "@docusaurus/module-type-aliases": "^2.0.0-beta.3",
+    "@tsconfig/docusaurus": "^1.0.2",
+    "@types/react": "^17.0.14",
+    "@types/react-helmet": "^6.1.2",
+    "@types/react-router-dom": "^5.1.8",
+    "@typescript-eslint/eslint-plugin": "^4.28.3",
+    "@typescript-eslint/parser": "^4.28.3",
+    "cross-env": "^7.0.3",
+    "docusaurus-plugin-sass": "^0.2.1",
+    "esbuild": "^0.12.15",
+    "esbuild-loader": "^2.13.1",
+    "eslint": "^7.30.0",
```

Comment:
We have these deps, eslint, prettier and typescript already on the root, but we can clean up that later.

Reply:
Done :)